src/App.js

```diff
@@ -22,34 +19,31 @@ function App() {
 
   // if (error) console.log(error);
   // for Get user------------------------------------------>
-
-  useEffect(()=>{
-    getData();
-  }, [])
   const [data, setData] = useState([]);
-  const [loading, setLoading]=useState(false);
-
-   const getData = async ()=>{
-
+  const [loading, setLoading] = useState(false);
 
+   const getData = async () => {
     try{
      const res = await axios.get("https://randomuser.me/api?results=1")
      setData(res.data.results)
      setLoading(true)
-    }catch(err){
-      console.log(err.message)
+    } catch(err) {
+      console.error(err.message)
     }
    }
-  return (
    
+  useEffect(() => {
+    getData();
+  }, []);
+  
+  return (
     <>
-    
      <Header/>
       <div className="container">
         <ul>
           {loading &&
```

Comment:
My assumption here is that you'd have a Loading spinner or something to show that the data is being fetched.

Reply:
I think this should be `contact.length &&` instead of `loading &&`